x-pack/plugins/security_solution/server/lib/timeline/routes/notes/delete_note.ts

```diff
@@ -40,27 +40,25 @@ export const deleteNoteRoute = (router: SecuritySolutionPluginRouter, config: Co
 
         try {
           const frameworkRequest = await buildFrameworkRequest(context, request);
-          const noteId = request.body?.noteId ?? '';
-          const noteIds = request.body?.noteIds ?? null;
-          if (noteIds != null) {
-            await deleteNote({
-              request: frameworkRequest,
-              noteIds,
-            });
+          if (!request.body) {
```

Comment:
@kqualters-elastic can we just update this api to only take `noteIds` rather than needing `noteId` as well?

Reply:
@michaelolo24 @kqualters-elastic We should add that as a breaking change for the next major release